### ISSUE TITLE
Show commit dates

### ISSUE THREAD

**Issue by Kissaki** [Kissaki/ChangelogTransform#8]
`git log --pretty=raw`

**Comment by ZingBlue** [comment]
Do you still want me to work on this?

**Comment by Kissaki** [comment]
If you want and feel comfortable with it, sure. Just tell me which one you are working on and when.

I just felt like the easy task of categorization is finished now.

For this one git log can use other formats than `--oneline` to include the date, and then that can be added to the `Commit` and consequently potentially to the grouped item.

**Comment by Kissaki** [comment]
See https://git-scm.com/docs/pretty-formats

**Comment by ZingBlue** [comment]
> 
> 
> See https://git-scm.com/docs/pretty-formats

@Kissaki I've been working on the project for about the last 30 minutes now. I figured out how to add text to the HTML file and some general things. But I can't seem to find the file that contains the Git arguments. Could you point me to it? I think I could figure it out from there. I am not great at looking through files and this project has a lack of comments.

**Comment by Kissaki** [comment]
[GitLog.cs#L20](https://github.com/Kissaki/ChangelogTransform/blob/master/ChangelogTransform/GitLog.cs#L20)

**Comment by ZingBlue** [comment]
> 
> 
> [GitLog.cs#L20](https://github.com/Kissaki/ChangelogTransform/blob/master/ChangelogTransform/GitLog.cs#L20)

Huh, that's odd I tried that before and it did not change the change notes. Also, how do you want the time? I.e 24hr clock, 12 hr clock, what timezone? etc...

**Comment by Kissaki** [comment]
Just the date. Maybe `yyyy-MM-dd` to stay international.

**Comment by ZingBlue** [comment]
> 
> 
> Just the date. Maybe `yyyy-MM-dd` to stay international.

Will do! (or at least try.)

**Comment by Hayden4126** [comment]
@Kissaki What exactly does line 20 do? If I make the string blank nothing changes. or if I add anything to it.

**Comment by ZingBlue** [comment]
@Kissaki Why does line 20 seem to not do anything?

**Comment by Kissaki** [comment]
It is used in [L57](https://github.com/Kissaki/ChangelogTransform/blob/master/ChangelogTransform/GitLog.cs#L57) as parameters to the git executable.

Make sure the RepoPath is an existing clone of the mumble repository on your PC https://github.com/Kissaki/ChangelogTransform/blob/master/ChangelogTransform/Program.cs#L14

**Comment by ZingBlue** [comment]
> It is used in [L57](https://github.com/Kissaki/ChangelogTransform/blob/master/ChangelogTransform/GitLog.cs#L57) as parameters to the git executable.
> 
> Make sure the RepoPath is an existing clone of the mumble repository on your PC https://github.com/Kissaki/ChangelogTransform/blob/master/ChangelogTransform/Program.cs#L14

@Kissaki Yeah, I did this awhile back. The string just sleems to do nothing, I put "testing lololol" in it and removed the current args, recompiled reran the HTML file, refreshed(to double check) and nothing changed.

**Comment by Kissaki** [comment]
Ah sorry, it stores the result of the call into `Store_History` and if that file exists it loads from there, not from Git.

As you can see here https://github.com/Kissaki/ChangelogTransform/blob/master/ChangelogTransform/GitLog.cs#L32-L41

So if you remove that cache file it should read from git again.

**Comment by ZingBlue** [comment]
@Kissaki Date or commit message first?

**Comment by ZingBlue** [comment]
I have created a pull request that should fix this issue.
